elytron/src/test/java/org/wildfly/extension/elytron/CredentialStoreUpdatesTestCase.java

```diff
@@ -503,6 +496,77 @@ public void testFailedRuntimeOperationWithClearTextAttributeOnly() throws Except
         }
     }
 
+    @Test
+    public void testAddOperationWithCredentialReferenceWithStoreOnly() throws Exception {
+        testAddKeyStoreWithInvalidCredentialReference(KS_NAME, NON_EMPTY_CS_NAME, null, null);
+    }
+
+    @Test
+    public void testAddOperationWithCredentialReferenceWithAliasOnly() throws Exception {
+        testAddKeyStoreWithInvalidCredentialReference(KS_NAME, null, EXISTING_ALIAS, null);
+    }
+
+    @Test
+    public void testAddOperationWithCredentialReferenceWithClearTextOnly() throws Exception {
+        try {
+            String password = "secret";
+            addKeyStoreWithCredentialReference(KS_NAME, null, null, "secret", false, false);
```

Comment:
We could use `password` variable instead of `"secret"` string.

Reply:
Fixed.